speakers.js

```diff
@@ -0,0 +1,50 @@
+function Speaker(name, profession, description, featureImg) {
+  this.name = name;
+  this.profession = profession;
+  this.description = description;
+  this.featureImg = featureImg;
+}
+
+const speakers = [];
+
+function createSpeakers() {
+  let speaker;
+
+  speaker = new Speaker('Bill Gates', 'Software developer, investor, entrepreneur', 'William Henry Gates III is an American business magnate, software developer, investor, author, and philanthropist.', 'assets/billgates.jpeg');
+  speakers.push(speaker);
+
+  speaker = new Speaker('Elon Musk', 'Founder, CEO and Chief Engineer of SpaceX', 'Elon Reeve Musk is an entrepreneur and business magnate. He is the founder, CEO, and Chief Engineer at SpaceX; early stage investor, CEO, and Product Architect of Tesla, Inc.', 'assets/elonmusk.png');
+  speakers.push(speaker);
+}
```

Comment:
Add more speakers  so you could enable this  ![wrong2](https://user-images.githubusercontent.com/58376020/125110796-5ac62600-e0e5-11eb-8333-fbe8bbc74c73.png) button